api-docker.env

```diff
@@ -0,0 +1,9 @@
+export PGHOST=localhost
+export PGPORT=5432
+export PGDATABASE=census
+export PGUSER=insights
+export PGPASSWORD=insights
+export POSTGRES_PASSWORD=mylocalsecret
+export PGHOST_INTERNAL=host.docker.internal
```

Comment:
I know this is a trivial point but does this need to be a env var since it's really a docker constant?

(my justification for querying this is we have a lot of env vars and maybe reducing by one is at least one other thing not to worry about)